docs/guide/upgrading.md

```diff
@@ -1,13 +0,0 @@
-# Upgrading
```

Comment:
Eek! We need this file -- it shows up in the dbt output if your version is out-of-date:

```bash
$ dbt --verison
installed version: 0.6.0
   latest version: 0.6.1
Your version of dbt is out of date! You can find instructions for upgrading here:
http://dbt.readthedocs.io/en/master/guide/upgrading/
```